test/toolbox_image.sh

```diff
@@ -288,203 +110,88 @@ run_test()
 
 test_execute_busybox()
 {
-    chroot "${rootfs_dir}" /bin/busybox true
+    chroot "${toolbox_root_dir}" /bin/busybox true || return 1
 }
 
 test_execute_sha512()
 {
-    chroot "${rootfs_dir}" sha512sum /bin/busybox > "${rootfs_dir}${TMP_TEST_IMAGE_FILE}"
-    chroot "${rootfs_dir}" sha512sum -csw "${TMP_TEST_IMAGE_FILE}"
+    chroot "${toolbox_root_dir}" sha512sum /bin/busybox > "${toolbox_root_dir}${TMP_TEST_IMAGE_FILE}" || return 1
+    chroot "${toolbox_root_dir}" sha512sum -csw "${TMP_TEST_IMAGE_FILE}" || return 1
 }
 
 test_execute_sfdisk()
 {
-    chroot "${rootfs_dir}" /sbin/sfdisk -l "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
+    chroot "${toolbox_root_dir}" /sbin/sfdisk -l "${TMP_TEST_IMAGE_FILE}" || return 1
 }
 
 test_execute_mkfs_ext4()
 {
-    chroot "${rootfs_dir}" /sbin/mkfs.ext4 "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
-    chroot "${rootfs_dir}" /sbin/fsck.ext4 -fn "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
+    chroot "${toolbox_root_dir}" /sbin/mkfs.ext4 "${TMP_TEST_IMAGE_FILE}" || return 1
+    chroot "${toolbox_root_dir}" /sbin/fsck.ext4 -fn "${TMP_TEST_IMAGE_FILE}" || return 1
 }
 
 test_execute_resize2fs()
 {
-    test_execute_mkfs_ext4
-    chroot "${rootfs_dir}" /usr/sbin/resize2fs "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
-    chroot "${rootfs_dir}" /sbin/fsck.ext4 -fn "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
+    test_execute_mkfs_ext4 || return 1
+    chroot "${toolbox_root_dir}" /usr/sbin/resize2fs "${TMP_TEST_IMAGE_FILE}" || return 1
+    chroot "${toolbox_root_dir}" /sbin/fsck.ext4 -fn "${TMP_TEST_IMAGE_FILE}" || return 1
 }
 
 test_execute_mkfs_f2fs()
 {
-    chroot "${rootfs_dir}" /usr/sbin/mkfs.f2fs "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
-    chroot "${rootfs_dir}" /usr/sbin/fsck.f2fs "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
+    chroot "${toolbox_root_dir}" /usr/sbin/mkfs.f2fs "${TMP_TEST_IMAGE_FILE}" || return 1
+    chroot "${toolbox_root_dir}" /usr/sbin/fsck.f2fs "${TMP_TEST_IMAGE_FILE}" || return 1
 }
 
 test_execute_resizef2fs()
 {
-    test_execute_mkfs_f2fs
-    chroot "${rootfs_dir}" /usr/sbin/resize.f2fs "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
-    chroot "${rootfs_dir}" /usr/sbin/fsck.f2fs "${TMP_TEST_IMAGE_FILE}" 1> /dev/null
+    test_execute_mkfs_f2fs || return 1
+    chroot "${toolbox_root_dir}" /usr/sbin/resize.f2fs "${TMP_TEST_IMAGE_FILE}" || return 1
+    chroot "${toolbox_root_dir}" /usr/sbin/fsck.f2fs "${TMP_TEST_IMAGE_FILE}" || return 1
 }
 
 test_execute_mount()
 {
-   chroot "${rootfs_dir}" /bin/mount --version 1> /dev/null
+   chroot "${toolbox_root_dir}" /bin/mount --version
```

Comment:
I think you want to add `|| return 1` here

Reply:
Yeah for documentation purpose and consistency, although not really required because it the only statement in the function.